posts/napari-cosmx-basics/using-napari-cosmx.qmd

```diff
@@ -0,0 +1,763 @@
+            ├── FovTracking
+            ├── Morphology_ChannelID_Dictionary.txt
+            ├── Run1000_20230406_205644_S1_Beta12_ExptConfig.txt
+            ├── Run1000_20230406_205644_S1_Beta12_SpatialBC_Metrics4D.csv
+            ├── Shading
+            ├── c902.fovs.csv
+            └── latest.fovs.csv
+```
+
+### Expected Raw Data Format {#sec-expected-raw-format}
+
+In order for `napari-cosmx` to stitch this non-AtoMx example dataset, we'll need to 
+rearrange the folders so that the nested raw data are at the top level.
+After rearrangement, the proper file structure should look like this:
+
+```{default filename="Terminal"}
+#| eval: false
+
+tree -L 2
+
+.
+├── AnalysisResults
+│   └── cp7bjyp7pm
+├── CellStatsDir
+│   ├── CellComposite
+│   ├── CellOverlay
+│   ├── FOV001
+│   ├── FOV002
+...
+│   ├── Morphology2D
+│   └── RnD
+└── RunSummary
+    ├── Beta12_Affine_Transform_20221103.csv
+    ├── FovTracking
+    ├── Morphology_ChannelID_Dictionary.txt
+    ├── Run1000_20230406_205644_S1_Beta12_ExptConfig.txt
+    ├── Run1000_20230406_205644_S1_Beta12_SpatialBC_Metrics4D.csv
+    ├── Shading
+    ├── c902.fovs.csv
+    └── latest.fovs.csv
+```
+
+There are a few ways to rearrange. The first method retains the original folder
+structure and simply makes symbolic links to the data in the expected format. Here's how
+to do it in unix/mac (Windows not shown).
+
+```{default filename="Terminal"}
+#| eval: false
+# Terminal in Mac/Linux
+
+# cd to folder containing HalfBrain. Then, 
+
+mkdir -p RawFiles && cd $_
+ln -s ../HalfBrain/AnalysisResult/HalfBrain_20230406_205644_S1/AnalysisResults .
+ln -s ../HalfBrain/CellStatsDir/HalfBrain_20230406_205644_S1/CellStatsDir .
+ln -s ../HalfBrain/RunSummary/HalfBrain_20230406_205644_S1/RunSummary .
+
+```
+
+Alternatively, we could manually move folders.
+Specifically, in your Finder window, create a folder named `RawData`. Then, move:
+
+- `HalfBrain/AnalysisResult/HalfBrain_20230406_205644_S1/AnalysisResults` to `RawData/AnalysisResults`
+- `HalfBrain/CellStatsDir/HalfBrain_20230406_205644_S1/CellStatsDir` to `RawData/CellStatsDir`
+- `HalfBrain/RunSummary/HalfBrain_20230406_205644_S1/RunSummary` to `RawData/RunSummary`
+
+Once the file structure is properly formatted, use the [stitching widget method](../using-napari-for-cosmx-data/index.qmd#sec-stitch-images){target="_blank"}
+for an earlier blog post to create the mouse brain napari files. 
+
+<!-- For this post, I named my output folder `mouse_brain_napari_files`. -->
+
+### Adding metadata {#sec-adding-metadata}
+
+We will also use the cell typing data from the Seurat file. Let's include the
+following metadata columns: 
+
+- `RNA_nbclust_clusters`: the cell typing results (with abbreviated names)
+- `RNA_nbclust_clusters_long`: (optional) human-readable cell type names
+- `spatialClusteringAssignments`: spatial niche assignments
+
+Note that the Seurat file contains _both_ tissues of mouse brain dataaset. We need to filter
```

Comment:
better to say "two sections of mouse brain samples".

Reply:
Done: e730ffa130bec04f6ede63645e2c715ad72bd319